packages/wrangler/src/type-generation.ts

```diff
@@ -165,7 +291,7 @@ function writeDTSFile({
 				.includes("Generated by Wrangler")
 		) {
 			throw new UserError(
-				"A non-wrangler worker-configuration.d.ts already exists, please rename and try again."
+				`A non-wrangler ${path} already exists, please rename and try again.`
```

Comment:
ASIDE: it feels to me like a common usage would be to run this `wrangler types` command quite often, particularly every time a change is made to `wrangler.toml` or `.dev.vars`. This error requires them to delete the old file first, right?
I wonder if we could offer some `--force` option that will always overwrite the file without an error.

Reply:
Fortunately the implementation already thought of this use case, and there is a condition above this `throw` that checks if the dts file was `Generated by Wrangler` (meaning it contains this string). Can confirm this works nicely, we regenerate the file regularly in CI to check if it's consistent with wrangler config.

Reply:
Oh sweet! Thanks for confirming.